host/traceh.c

```diff
@@ -67,88 +71,152 @@ void initialize_log_config()
         // inititalize if not already
         _log_level = _env2log_level();
         _log_file_name = getenv("OE_LOG_DEVICE");
+        _custom_log_format = getenv("OE_LOG_FORMAT");
+        _json_escape = getenv("OE_JSON_ESCAPE");
+        if (_custom_log_format)
```

Comment:
I am wondering what it's called "OE_JSON_ESCAPE" instead of OE_LOG_ESCAPE.
I think escaping could apply to type pf message contents.
Why is JSON tied to it?

Reply:
Changed to `OE_LOG_ESCAPE`. Originally tied to JSON as I was looking at the JSON standard to implement string escaping